proxies/live/apiproxy/proxies/default.xml

```diff
@@ -38,6 +38,10 @@
       (proxy.pathsuffix MatchesPath "/_ping") and ((request.verb = "GET") or (request.verb = "HEAD"))
     </Condition>
   </RouteRule>
+  <RouteRule name="sync-wrap">
+    <Condition>request.verb == "GET" and request.header.prefer != "respond-async" and request.header.x-sync-wrapped != "true"</Condition>
```

Comment:
I think we need this to be POST, not GET

Reply:
Might also need to filter to just the $process-message endpoint, as that's the one which will respond asynchronously

Reply:
Yeah had some questions on this as there's some conflicting documentation for using it. I just wanted to test out communications with sync-wrap initially, but should move it over to process message

Reply:
Removed to keep refactoring separate